src/main/java/com/grum/geocalc/Point.java

```diff
@@ -47,6 +48,27 @@ public Point(Coordinate latitude, Coordinate longitude) {
         this.latitude = latitude.getValue();
         this.longitude = longitude.getValue();
     }
+    
+    
+    /**
+     * Build new point with latitude and longitude
+     * @param latitude
+     * @param longitude
+     * @return 
+     */
+    public static Point build(Coordinate latitude, Coordinate longitude) {
+        return new Point(latitude, longitude);
+    }
+    
+    /**
+     * Build new point with latitude and longitude
+     * @param latitude
+     * @param longitude
+     * @return Point
+     */
+    public static Point build(double latitude, double longitude) {
+        return build(DegreeCoordinate.build(latitude), DegreeCoordinate.build(longitude));
```

Comment:
I would object here as well. It looses the type-safety of what a double value means (ie is it degrees, radians ?). The javadoc could indicate it but it currently does not